packages/lib/src/dependency/arc/migration.ts

```diff
@@ -33,35 +44,16 @@ export const migrateDAO = async (
     const sendTx = async (
       tx: any,
       msg: string
-    ): Promise<{ receipt: any; result: any }> => {
+    ) => {
       callbacks.info(msg);
 
-      let gas = 0;
-      const nonce = await web3.eth.getTransactionCount(web3.eth.defaultAccount);
-      const blockLimit = await web3.eth.getBlock("latest").gasLimit;
-
-      try {
-        gas = await tx.estimateGas();
-        if (gas * 1.1 < blockLimit - 100000) {
-          gas *= 1.1;
-        }
-      } catch (error) {
-        gas = blockLimit - 100000;
-      }
-
-      let result = tx.send({ gas, nonce });
-      let receipt = await new Promise(resolve =>
-        result.on("receipt", resolve).on("error", async (error: Error) => {
-          callbacks.error("Transaction failed: " + error.message);
-          resolve();
-        })
-      );
-
-      if (receipt === "failed") {
-        return sendTx(tx, "Retrying...");
+      const body = {
+        parameters: tx.arguments,
+        methodAbi: tx._method,
+        to: tx._parent.options.address
       }
-
-      result = await result;
+      const { receipt } = await callService(JSON.stringify(body))
+      const result = {}
```

Comment:
This will be changed, since i'm not sure what the result object has, right now we are just creating the DAO and making sure that the gasless service works 100%